CONTRIBUTING.md

```diff
@@ -30,7 +30,47 @@ conform enforce --commit-msg-file $1
 EOF
 ```
 
+In addition, all commits should be signed by the commiter.
```

Comment:
We should probably add `git commit -s` here.

Reply:
Agree. Perhaps even spell out that this is different than GPG signing. That has already confused a number of people.